### PR TITLE
PR 16005 changelog: added to API.

### PR DESCRIPTION
**Pull request by orionlee**
### Description

This pull request is a follow up of PR #16005 .

It added a changelog entry to API as `16005.api.rst` . It is almost identical to the existing [`16005.bugfix.rst`](https://github.com/astropy/astropy/blob/0e216130eafb23547bc956534ce412c9a68e1fa8/docs/changes/io.ascii/16005.bugfix.rst), except it does not contain the text about fixing IPAC table reading.

@taldcroft @mvhk @dhomeier Please review.

- [ ] By checking this box, the PR author has requested that maintainers do **NOT** use the "Squash and Merge" button. Maintainers should respect this when possible; however, the final decision is at the discretion of the maintainer that merges the PR.

### CODE REVIEW

File: docs/changes/io.ascii/16005.api
Comment (pllim) [inline docs/changes/io.ascii/16005.api:1]: Has to have `.rst` suffix. See other fragments as example. Thanks!

Comment (orionlee) [reply docs/changes/io.ascii/16005.api:1]: Fixed. Somehow the editor's "save as" lost the extension!

**Comment by pllim** [comment]
Thanks! I'll let the maintainers vet the wordings.

**Review by dhomeier — APPROVED** [review]
As noted I don't have a strong opinion as to how much #16005 constitutes a bugfix; this addition looks good to me – only other option I see here is whether `16005.bugfix.rst` should be shortened to just the part about the IPAC failure, or on ensuring consistent behaviour across platforms.

**Review by mhvk — APPROVED** [review]
Happy with this - thanks, @orionlee!

**Comment by pllim** [comment]
> only other option I see here is whether 16005.bugfix.rst should be shortened to just the part about the IPAC failure, or on ensuring consistent behaviour across platforms.

@dhomeier , does this need to be addressed? You left this comment but then you approved anyway. Just want to make sure.

**Comment by dhomeier** [comment]
Just what has been discussed in https://github.com/astropy/astropy/pull/16005#issuecomment-1942072487 ff., I am not insisting on any particular wording.

**Comment by orionlee** [comment]
Added a commit to change the bugfix to be just about IPAC `long` column proble.

Given's mvhk's response https://github.com/astropy/astropy/pull/16005#issuecomment-1942074163 , I presume he believes the change has been (or will be) done.

**Comment by mhvk** [comment]
I just thought it was good enough as it was - but now it is even better!

**Comment by dhomeier** [comment]
Thanks @orionlee !

**Pull request merged**